Source/WebCore/rendering/svg/SVGResourcesCycleSolver.cpp

```diff
@@ -46,7 +46,7 @@ bool SVGResourcesCycleSolver::resourceContainsCycles(LegacyRenderSVGResourceCont
 
     RenderObject* node = &resource;
     while (node) {
-        if (node != &resource && node->isSVGResourceContainer()) {
+        if (node != &resource && node->isLegacySVGResourceContainer()) {
```

Comment:
Cycle solving is currently disabled for LBSE?

Reply:
Yes it is, and we're going to leave it disabled -- SVG resources are being reworked in parallel.